img/Google Icon.svg

```diff
@@ -0,0 +1,9 @@
vph/YBgEQCB6BzCNnoa+tvXre3cGLDhFlS8DKtiHaBYvA+3M3NB47ZcJvuXifFqYJTIxfpoJVIkQDAq4QyDxyZrp07VXznnDFAYx6RgA/tD1D7Z6j4bOrz2HiRzBAyz3GsAwCQSCAR85BqIJzMUCAnWPpq6XMAK1m/hmzusrXQOAcBEDAcQIY5ew40kAYhAAHogzOBZGZM6zpXiIud84qLIEACPhGQGSFpfm6VZ/53Zu+xQDHrhDAbkiuYPXPaO2MeY9JioYIyW/8iwKeQQAECiVg7npJy1fXvDF0HMS3UJrB7I874GDWxZGoqmZVX0KK7sOmDo7ghBEQ8IyAkLxCyrqmdsbcVz1zCkeeE8AdsOfIvXNYe9W8p/fzMYPNetJmWzLvPMMTCIBAXgREUpn133dVjIT45kUwVJ1wBxyqcuUf7IGR0vQLJj49fyvoCQIg4BYBc9cryvoCVrRyi3Dw7OIOOHg1cSWitVfOW0m7KoaRyHcy75Zc8QKjIAACORMQ2UtE36BdFSMhvjnTC3UH3AGHunz5BT9szvQhSuRuFrkwPwvoBQIg4AQBM69XaftLL1715N+dsAcb4SIAAQ5XvRyNduTj1RdoobvxWNpRrDAGAl0SEKK3SOQOM06jy8ZoEFkCeAQd2dJ2ndiLV8xbYB5La5KbmHRD1z3QAgRAoBACmdc/It8p3V96JsS3EJLR6Is74GjUseAsRs2b2le38rdJ6GZiThZsEAZAAAQOIyAsT1tszVw9Y+67h53Al9gSgADHtvTtJ141Z/qZJPIjvB9unw+OgkCuBIRkPQl9FXe8uZKLfnsIcPRrnFeGw2dVT2Wm7+H9cF740AkESFj+QcTfqN145iNUU4N5+LgmjiIAAT4KCQ58SKCmRlUNem0akf4uEw/68Dg+gAAIdEJAGknTD3u2lP1kyeceau6kIU7FnAAEOOYXQFbpfyjE8gNseZgVMTSKIwGRvZrpXl2sf/jy1N83xhEBcs6NAAQ4N16xbl11/41JKt9+NQl9A0Ic60sBybchkFnmVeQJy0r8BwZYtQGDj10SgAB3iQgNjiRghFiVNtygLfoaNno4kg6+x4aASEqYZiVs/k/sVhSbqjuaKATYUZzxMnb+r68v2V3cdAMp/jITDYxX9sg2tgTM0pFMv2Kx/xsrWMX2KnAkcQiwIxhjbsS8Iz5t3UVM9HViHhNzGkg/qgQOCq+k6Ie118zbEtU0kZd3BCDA3rGOhafhc6ZXsba/SESfZWKstBaLqkc7SbNKnAjfmy7Rd2NwVbRr7XV2EGCvicfE3+hHp52aVvSvTHINMfeISdpIM0IEhOR1pemeHi1lv8J0oggVNkCpQIADVIwohnJgiUt1s7DcjAFbUaxwtHLKjGhmelIR/Wz1jHkLmUmilSGyCRIBCHCQqhHlWGpq1MjB6yZpLTcw8VSsNx3lYocvNyGuE9IPJ5R1H6YSha9+YY0YAhzWyoU47qpHqgewRdeI4hswnzjEhYxC6CJrNNODZfvLfoPHzFEoaLhygACHq17RihZ3xdGqZ0iyyQyqIp6llfXw2hlza0MSNsKMIAEIcASLGsaUzLvidDNfw4quZ+KzwpgDYg4ugcw+vCR/IOLfUFPFs7U3PZAKbrSILC4EIMBxqXSI8hw1Z/rAdNq+jFmqmdX4EIWOUANE4OASkctEeF6iRB5ZXT1/e4DCQyggQBBgXASBJjBszvQhlm3PEKbLsTVioEsVmOCE5BXW9BizPQsrVQWmLAikHQIQ4Hag4FAwCbQR4yuwPWIwa+RHVIfudFn4KUt4PtZl9qMK8JkPAQhwPtTQx3cCVXOmn8lp+yJS/AkSGYdpTb6XxNsARPaKomeJ+EkrqZ/C42Vv8cObMwQgwM5whBUfCQyZc2vPEnvrhUw8hZg+jo0hfCyGi66F6F0SWURE80ubyxZi2pCLsGHaEwIQYE8ww4mXBDKDuLR9IYtMZuJLsBSml/Sd82WmC2mhRcK8UJS19KUZczc4Zx2WQMB/AhBg/2uACFwkYLZMbCrZM1KxTBCWMSx0LhGXu+gSpvMmII3CtJhsWkwJa8ma6XNfw1KQecNExxAQgACHoEgI0UECNTVq2JD1g1jb45gyYjwWq3E5yDdLU5mBU0QbiahWaVplJ6xVazec/hLV1OgsTaAZCISeAAQ49CVEAoUSMEtjSoLOIaZzmNUwIj0YG0cUSvXw/sLyDyJ6kW1ayRat3cvHrNww42d7Dm+FbyAQLwIQ4HjVG9lmScCszCUpHqZtGkxMZzGRWZ1rMN4ndwHQbFpPtFGYNrCm9cK0ntK0FhvYd8ENp2NJAAIcy7Ij6bwI1NSo0ae+ekpKyRmKeYiIDGTmk4XoJBY5Lk5ToczuQST234jUehF5w7LoNdL2G1j4Iq8rC51iSgACHNPCI22HCZiNJU576SM2JU9iJSezloHCNJBEBhKr41ikX2juns0cW6ZNZP7TtJWJ/iGK31Uk76s0v9ettfTvmALk8PUDc7EkAAGOZdmRtB8Equ6/MUndGyqo2KoQW8qJdAUxVSriSk1SxkTHEFNfIlXEIgkSKTVxClM3Iu5uPjNJ70zsQslDgm42GmCm5oNt95DwwY0GpImFDnxm3i0sjaRphzBtV8RNmqSeSe1UrHfabO1MEO3YQ5X1eDebIYz/gQAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIgAAIOEjg/wN/ChVIgYWaEQAAAABJRU5ErkJggg=="/>
```

Comment:
Coś się źle wyeksportowało z Figmy - w tej chwili jest to plik SVG z osadzonym plikiem PNG. ;) Później sprawdzę o co biega z tym pliczkiem i podeślę Ci na FB